modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormValuesConverterUtil.java

```diff
@@ -40,8 +40,9 @@
 				List<DDMFormFieldValue> nestedDDMFormFieldValues =
 					new ArrayList<>();
 
-				if (StringUtil.equals(ddmFormField.getType(),
-					DDMFormFieldTypeConstants.FIELDSET)) {
+				if (StringUtil.equals(
+						ddmFormField.getType(),
+						DDMFormFieldTypeConstants.FIELDSET)) {
```

Comment:
@mateussgn, ideally, this SF should be on the previous commit or on a dedicated one to make it clear to the reviewers and also consistent with the commit message. :wink: